app/controllers/etds_controller.rb

```diff
@@ -21,14 +21,21 @@ def index
 
   # POST /etds
   # receives xml data from peoplesoft
-  def create
+  def create # rubocop:disable Metrics/AbcSize
     logger.info("RAW XML: #{request.raw_post}")
 
     return unless submission
 
-    @submission.update!(submission_attributes)
-    peoplesoft_actions
-    render_created
+    Submission.transaction do
```

Comment:
Since we have a few different changes happening in one controller action, I thought it wise to wrap them all in a transaction so things don't get in a weird state. I could be convinced not to worry about this if someone has a compelling argument.